tests/unit/api_image_test.py

```diff
@@ -271,6 +272,28 @@ def test_push_image_with_auth(self):
             timeout=DEFAULT_TIMEOUT_SECONDS
         )
 
+    @requires_api_version('1.46')
+    def test_push_image_with_platform(self):
```

Comment:
Not to be picky, but your function can take either Platform OR dict, but you are just testing one.